tools/main.py

```diff
@@ -58,20 +135,38 @@ def main():
     num_params, flops = compute_model_complexity(model, (1, 3, cfg.data.height, cfg.data.width))
     print('Main model complexity: params={:,} flops={:,}'.format(num_params, flops))
 
+    if cfg.model.resume and check_isfile(cfg.model.resume):
+        cfg.train.start_epoch = resume_from_checkpoint(
+            cfg.model.resume, model, optimizer=optimizer, scheduler=scheduler
```

Comment:
Optimizer and scheduler should be created in advance in case of resume

Reply:
Ok, fixing it

Reply:
Fixed